src/pages/app/navigation-bar/control-button.ts

```diff
@@ -1,14 +1,23 @@
 import h from "hyperscript"
+import cx from "classnames"
 
 type ControlButtonProps = {
-  label: string
+  label: Node | string
   image: string
-}
-export const ControlButton = ({ label, image }: ControlButtonProps) => {
+  className?: string
+} & Partial<HTMLButtonElement>
+
+export const ControlButton = ({
+  label,
+  image,
+  className,
+  type = "button",
+  ...props
+}: ControlButtonProps) => {
   return h(
     "button",
-    { className: "control-button" },
+    { className: cx("control-button", className), ...props },
```

Comment:
accept a custom class, spill all props passed as arguments to attributes of the html button